scripts/ci/upload-release

```diff
@@ -32,8 +32,8 @@ PKG_TARGET=$(python -c "import sys, json; \
   dcosrc = json.load(open('$PROJECT_ROOT/.dcosrc')); \
   sys.stdout.write(str(dcosrc['base']));")
 
-# release version (after switching to semantic-release, we cant use package.json anymore)
-PKG_VERSION="$(git tag -l --sort=-version:refname | grep ${PKG_TARGET} | head -1 | cut -d'+' -f2)"
+# next release version (passed in by semantic-release)
+PKG_VERSION="${1}"
```

Comment:
version is now injected by semanric-release ->  https://github.com/dcos/dcos-ui/pull/3192/files#diff-b78e9a225940bc2471e85d5d79220450R9